UM/Settings/Profile.py

```diff
@@ -30,7 +33,13 @@ def __init__(self, machine_manager, read_only = False):
         super().__init__()
         self._machine_manager = machine_manager
         self._changed_settings = {}
-        self._name = "Unknown Profile"
+        self._changed_settings_defaults = {}
```

Comment:
The working profile carries a "copy" of the profile it was based on, so the reset value widget can reset to the value set in the profile the user selected. Otherwise we can only reset back to the machine default. See CURA-626